app/experimenter/experiments/models/legacy.py

```diff
@@ -1287,9 +1287,7 @@ class ExperimentComment(ExperimentConstants, models.Model):
 
     @property
     def get_absolute_url(self):
-        return "{comment.experiment.experiment_url}/#comment{comment.id}".format(
-            comment=self
-        )
+        return urljoin(self.experiment.experiment_url, "#/comment{id}".format(id=self.id))
```

Comment:
is `#/` right?

The url should look like `experimenter/experiments/experiment-slug/#comment123`?

Maybe it'd be clearer if there was a test on this method?

Reply:
you're right, will fix

Reply:
If urljoin strips out the # then don't use urljoin at all just format string it `f"{self.experiment.experiment_url}#comment{id}`